content/cybersecurity/concepts/malware/terms/spyware/spyware.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'Spyware'
+Description: 'Spyware is malicious software designed to gather personal data from a device in order to send it to other third-party apps or companies without consent from the user.'
+Subjects:
+  - 'Cybersecurity'
+  - 'Computer Science'
+Tags:
+  - 'Cybersecurity'
+  - 'Computer Science'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/cyber-resilience-risk-management'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+
+**Spyware** is malicious computer software designed to gather personal data from a device in order to send it to other third-party apps or companies without the consent or knowledge of the user.
+
+## Why Is Spyware Harmful?
+
+Since Spyware steals a person's information and sends it others, it can put the person in all sorts of situations whether it's receiving bills for unintended purchases, private information being leaked to the public, or prevent you from signing in to the user's Apple or Google Play account to download music and apps. In other words, spyware ruins lives.
```

Comment:
```suggestion
Since Spyware steals a person's information and sends it others, it can put the person in all sorts of situations; whether it's receiving bills for unintended purchases, private information being leaked to the public, or preventing signing in to the user's Apple or Google Play account to download music and apps. In other words, spyware ruins lives.
```